src/openzaak/utils/tests/test_cache.py

```diff
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2022 Dimpact
+from django.test import SimpleTestCase
+
+import requests
+
+from openzaak.utils.cache import DjangoRequestsCache, requests_cache_enabled
+
+
+class DjangoRequestsCacheTests(SimpleTestCase):
+    def test_backend(self):
+        with requests_cache_enabled():
+            backend = getattr(requests.Session(), "cache", None)
```

Comment:
This only tests that the regular `requests.Session` has the correct backend, but since the issue was specifically with `vng_api_common.client.Client` (based on `ape_pie.APIClient`), let's verify that that works here.

You could do the same as shown in the traceback of the taiga issue (`client = get_client(self.selectielijst_procestype, raise_exceptions=True)`) in your test and then make assertions on the backend